src/EditorFeatures/CSharp/DecompiledSource/AssemblyResolver.cs

```diff
@@ -41,6 +41,11 @@ void BuildReferenceCache()
             }
         }
 
+        public bool IsGacAssembly(IAssemblyReference reference)
```

Comment:
@siegfriedpammer does this ever need to return `true` for our use case?

Reply:
This method is not called by decompiler - it does not care where the assemblies are from - only the `WholeProjectDecompiler` (not used by Roslyn) and ILSpy use this method. I would have it throw a `NotSupportedException` so that any bugs crop up faster.

Reply:
➡️ Now throws `NotSupportedException`